internal/server/query.go

```diff
@@ -23,4 +23,12 @@ const (
 		delete from worker_auth_ca_certificate
  		where state = @state;
 	`
+
+	authorizedWorkerQuery = `
+		select distinct w.worker_key_identifier 
+		from 
+			worker_auth_certificate_bundle as w
```

Comment:
I thought with the workers API you could delete a KMS worker which would disconnect it (although it could potentially auth again -- but maybe not if it's lost its KMS access). This query only handles PKI workers though right? Are KMS workers included in the rollup (seems like they should be) and would they end up being disconnected because of this?